demo/index.scss

```diff
@@ -11,3 +11,14 @@ $use-embedded-font: true;
 .container {
   margin: 20px;
 }
+
+
+//Example of how to style ws-tiles-chart using classes (.tile.groupName).
+.ws-tiles-chart .tile {
```

Comment:
Ah did you mean create a own SCSS demo file for the component and import it into index.scss? I got it :)